hasher-matcher-actioner/hmalib/lambdas/api/bank.py

```diff
@@ -305,7 +305,7 @@ def get_member(bank_member_id=None) -> PreviewableBankMemberWithSignals:
         )
 
     @bank_api.post("/update-bank-member/<bank_member_id>", apply=[jsoninator])
-    def update_bank_member(bank_member_id=None) -> Bank:
+    def update_bank_member(bank_member_id=None) -> BankMember:
```

Comment:
Weird little issue that caught my eye. Sometime I wonder, what is mypy doing if anything.